code/game/objects/items/weapons/tools/crowbar.dm

```diff
@@ -52,4 +52,37 @@
 	throw_range = 5
 	w_class = ITEM_SIZE_SMALL
 	matter = list(MATERIAL_STEEL = 150)
-	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked", "attacked", "slashed", "torn", "ripped", "cut")
\ No newline at end of file
+	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked", "attacked", "slashed", "torn", "ripped", "cut")
+
+/obj/item/crowbar/prybar/cheap
+	name = "discount pry bar"
+	desc = "A plastic bar with a wedge. It looks so poorly manufactured that you're sure it will break if you try to use it."
+	force = 2
+	throwforce = 4
+	matter = list(MATERIAL_PLASTIC = 60)
+	obj_flags = null
+	w_class = ITEM_SIZE_TINY
+	var/obj/item/material/shard/plastic/bits
+	
+	//List of things prybar has high chance of breaking on
+	var/list/break_chances = list(
+		/obj/machinery/door = 80,
+		/turf/simulated/floor/tiled = 25,
+		/mob/living = 15,
+		/obj/machinery = 15,
+		/atom = 05,
```

Comment:
You aren't ever going to be calling `afterattack` on something that isn't `/atom` - might be better to have some kind of default break chance? Will detail below.